hmpps-prisoner-search-indexer/applicationinsights.dev.json

```diff
@@ -16,16 +16,38 @@
   "sampling": {
     "percentage": 100,
     "overrides": [
+      {
+        "telemetryType": "dependency",
+        "attributes": [
+          {
+            "key": "server.address",
+            "value": ".+",
```

Comment:
we want to keep `InProc` `PUBLISH xxx` events at 100%, but want to reduce everything else.  Can't seem to find a good way to do it, so trying this way instead

Reply:
Hopefully you will eventually find a better way.